app/src/main/res/values/strings.xml

```diff
@@ -483,10 +483,10 @@
     <string name="low_glucose_values">Low Glucose Values</string>
     <string name="revert_to_default">Revert to Default</string>
     <string name="filtered_values">Filtered Values</string>
-    <string name="treatment_color">Treatment Color</string>
-    <string name="treatment_color_dark">Treatment Color Dark</string>
-    <string name="predictive_color">Predictive Color</string>
-    <string name="predictive_color_dark">Predictive Color Dark</string>
+    <string name="treatment_color">Insulin On Board Color</string>
+    <string name="treatment_color_dark">Insulin Activity Color</string>
+    <string name="predictive_color">Glucose Prediction Color</string>
+    <string name="predictive_color_dark">COB Prediction Color</string>
```

Comment:
It is true to say that these descriptions for the colors could help users in their understanding and that they are currently only used for these features afaik. If I need to add new graph element colors I can extend the preferences available as needed. Users have already accepted the disclaimer and warning when using the app for the first time so they are already aware of the limitations of the app and any predictive simulation features. I would accept these color naming string changes.

Reply:
Great, thanks!